core/ExceptionHandler.js

```diff
@@ -129,6 +129,10 @@ export class ExceptionHandler {
         const ret = Object.assign({}, options),
             isAutoRefresh = exception.fetchOptions && exception.fetchOptions.isAutoRefresh;
 
+        if (exception.name == 'AbortError') {
```

Comment:
These defaults are important, but should still not override what the user specified in handleException().

Also, can we change all of this code to use the much cleaner "withDefault" method?